light_curves/code_src/icecube_functions.py

```diff
@@ -16,7 +16,7 @@
 from data_structures import MultiIndexDFObject
 
 
-def icecube_get_lightcurve(sample_table, icecube_select_topN=3):
+def Icecube_get_lightcurve(sample_table, icecube_select_topN=3):
```

Comment:
It doesn't matter here, but I like the practice of making all optional arguments mandatory keyword argument. So in user code there won't be any semi-cryptic integers, as well as a more flexible way of extending the API with new arguments. (It doesn't matter here as there are only two arguments, but when a code is expected to evolve, then this practice is useful)